spec.bs

```diff
@@ -517,6 +536,34 @@ to conduct an auction to select an advertisement to display to the user, they ca
 tells the browser how to conduct the auction and which on-device recorded interests are allowed to
 bid in the auction for the chance to display their advertisement.
 
+<h3 id="createauctionnonce">createAuctionNonce()</h3>
+
+{{Window/navigator}}.{{Navigator/createAuctionNonce()}} creates an auction nonce, a one-time ID

```

Comment:
This paragraph seems like it should be a `Note`.

Reply:
great, it fits the goal perfectly. Thanks!